en/docs/develop/extend/myaccount/set-up-my-account-in-a-dev-environment.md

```diff
@@ -83,30 +83,31 @@ Follow the steps given below to set up My Account in a development environment.
 Fork the original repository to start working on it. You can also directly clone the original repo but it is
 recommended to create your own fork.
 
-```java
+```
 git clone https://github.com/wso2/identity-apps
 cd identity-apps
 ```
 
 Build the project using either of the following commands.
 
-```java
+```
 mvn clean install
 ```
 or 
 
-```java
+```
 npm run build
 ```
 
 ---
 
 ## Step 3: Run My Account in dev mode
 
-After the build is complete, navigate to the **My Account** directory and run the portal using the webpack dev server.
-```java
-cd apps/user-portal
+After the build is complete, navigate to the **myaccount** directory and run the application using the webpack dev server.
```

Comment:
Give the directory path

Reply:
how to navigate to the directory path is available in the command given below.